packages/office-ui-fabric-react/src/components/ContextualMenu/ContextualMenu.scss

```diff
@@ -19,7 +19,7 @@ $ContextualMenu-header-color: $menuHeaderColor;
 
 $ContextualMenu-divider-color: $menuDividerColor;
 
-$ContextualMenu-text-disabled: $disabledTextColor;
+$ContextualMenu-text-disabled: $menuTextDisabledColor;
```

Comment:
This is changing the color from $ms-color-neutralTertiary to $ms-color-neutralTertiaryAlt, is that purposeful?